test/elixir_analyzer/test_suite/accumulate_test.exs

```diff
@@ -8,7 +8,7 @@ defmodule ElixirAnalyzer.ExerciseTest.AccumulateTest do
       def accumulate([], _), do: []
 
       def accumulate([head | tail], fun) do
-        [fun.(head)] ++ accumulate(tail, fun)
+        [fun.(head) | accumulate(tail, fun)]
```

Comment:
That’s the exemplar solution I think… @angelikatyborska should we update it on the elixir repo as well?

Reply:
I'm not seeing anything that needs updating in the Elixir repo, it already uses `|`: https://github.com/exercism/elixir/blob/main/exercises/practice/accumulate/.meta/example.ex#L24

Anyway, remember about the difference between concept exercises and practice exercises. Only  concept exercises have _exemplar_ solutions. Practice exercises have _example_ solutions which can be less than ideal as long as they work 😁

Reply:
Oh I'm sorry, I just assumed. I didn't check because I was reviewing from my phone. I'll stop doing that, I think ^^

I know the difference between exemplar and example, I just didn't really remember which type `accumulate` was 😅
I should use a generic word... Maybe "exemploid"

Reply:
It's ok, don't top! If you can check yourself, it's better to check before complaining, but if you can't, it's better to complain and be wrong than not complain and let a problem slip. At least in my book.

I like "exemploid" :D sooner or later other maintainers will have the same issue with vocabulary I guess, how to talk about both exemplars and examples and be clear that you mean both...

Reply:
You really have a way with making people feel better about their mistakes, have I told you that? 💯 

And about  exemploids, careful of the [standard proliferation](https://xkcd.com/927/) ;)

Reply:
Anything to act on here? Just lurking around your conversation 👀

Reply:
Haha, sorry about that!
Nope, I think we are good, my initial comment was mistaken.